libs/components/modals/src/lib/modules/modal/modal.component.ts

```diff
@@ -96,10 +105,13 @@ export class SkyModalComponent implements AfterViewInit, OnDestroy {
   #hostService: SkyModalHostService;
   #elRef: ElementRef;
   #windowRef: SkyAppWindowRef;
+  #changeDetector = inject(ChangeDetectorRef);
```

Comment:
Put these properties that use `inject()` in their own block after all the other private property declarations (just above the constructor).

Reply:
Done